UK/vATIS/UK - Scottish AC.json

```diff
@@ -1,107 +1,132 @@
 {
-  "Name": "UK - Scottish AC",
-  "Composites": [
+  "name": "UK - Scottish AC",
+  "id": "b1431b6d-4010-48e4-bd25-3e0ae3a1d5fc",
+  "composites": [
     {
-      "Name": "Glasgow",
-      "Identifier": "EGPF",
-      "AtisType": 0,
+      "id": "b1f33d99-9432-4725-b99c-03b4cadda20d",
+      "name": "Glasgow",
+      "identifier": "EGPF",
+      "atisType": "Combined",
       "codeRange": {
         "low": "A",
         "high": "Z"
       },
-      "AtisFrequency": 29575,
-      "AtisVoice": {
-        "UseTextToSpeech": true,
-        "Voice": "UK Male"
+      "frequency": 129575000,
+      "atisVoice": {
+        "useTextToSpeech": true,
+        "voice": "UK Male"
       },
       "idsEndpoint": "",
       "useNotamPrefix": false,
       "useDecimalTerminology": true,
       "airportConditionsBeforeFreeText": false,
       "notamsBeforeFreeText": false,
-      "Presets": [
-        {
-          "Name": "RUNWAY 23 (SCO_CTR)",
-          "AirportConditions": "DEPARTING AIRCRAFT MAKE INITIAL CONTACT WITH SCOTTISH CONTROL ON 135.530",
-          "ArbitraryText": null,
-          "Template": "THIS IS [FACILITY] INFORMATION [ATIS_CODE] AT TIME [OBS_TIME]. AUTOMATIC. RUNWAY 23 IN USE. [TL]. [NOTAMS]. [ARPT_COND]. [WIND]. [VIS]. [RVR]. [PRESENT_WX]. [CLOUDS]. [TEMP]. [DEW]. [PRESSURE]. ACKNOWLEDGE RECEIPT OF INFORMATION [ATIS_CODE] AND ADVISE AIRCRAFT TYPE ON FIRST CONTACT."
+      "presets": [
+        {
+          "id": "62464d4d-16c2-48cf-a0f7-bca0f3c6bd59",
+          "name": "RUNWAY 23 (SCO_CTR)",
+          "airportConditions": "DEPARTING AIRCRAFT MAKE INITIAL CONTACT WITH SCOTTISH CONTROL ON 135.525",

```

Comment:
Is this the right frequency? The old one has 135.530 while this says 135.525 (seems to be the pre .833khz).